### PR TITLE
Adjusted some CLI commands in the readme

### PR DESCRIPTION
**Pull request by msdolbey**
Some of these commands didn't work for me when using the velero CLI.

After making these adjustments, they passed through correctly (namely, not having `[` or `]` in these different comma separated values).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -363,8 +363,8 @@ velero install \
     --plugins velero/velero-plugin-for-microsoft-azure:v1.5.0 \
     --bucket $BLOB_CONTAINER \
     --secret-file ./credentials-velero \
-    --backup-location-config resourceGroup=$AZURE_BACKUP_RESOURCE_GROUP,storageAccount=$AZURE_STORAGE_ACCOUNT_ID[,subscriptionId=$AZURE_BACKUP_SUBSCRIPTION_ID] \
```
Comment (ywk253100) [inline README.md:366]: @msdolbey Thanks for the contribution.

The parameters in `[]` mean that they are optional

Comment (msdolbey) [reply README.md:366]: Omg - thank you very much for pointing this out @ywk253100 

Mind if I add a note that these brackets mean that they are optional, or is that just a very well known thing and not necessary?

Comment (ywk253100) [reply README.md:366]: Yes, please add a note for these brackets

Comment (msdolbey) [reply README.md]: @ywk253100 thanks for the guidance - updated here

**Review by sseago — APPROVED** [review]

**Review by ywk253100 — APPROVED** [review]

**Comment by ywk253100** [comment]
@msdolbey Could you squash your commits and correct the PR number in the changelog file?

**Comment by jyriatntt** [comment]
Hi, can you please change the storageAccount=$AZURE_STORAGE_ACCOUNT_ID, it's incorrect, it should be $AZURE_STORAGE_ACCOUNT_NAME on velero install commands

**Comment by ywk253100** [comment]
I'm going to close this PR as inactivity, feel free to resolve the comment and reopen it again

**Pull request closed without merging**